src/emqx_sys_mon.erl

```diff
@@ -182,7 +182,15 @@ procinfo(Pid) ->
     case {emqx_vm:get_process_info(Pid), emqx_vm:get_process_gc_info(Pid)} of
         {undefined, _} -> undefined;
         {_, undefined} -> undefined;
-        {Info, GcInfo} -> Info ++ GcInfo
+        {Info, GcInfo} -> Info ++ GcInfo ++ get_proc_lib_initial_call(Pid)
```

Comment:
nit: prepend shorter list to longer is more efficient than appending short to long.